PromisesObjC.podspec

```diff
@@ -23,9 +23,14 @@ Pod::Spec.new do |s|
   s.public_header_files = "Sources/#{s.module_name}/include/**/*.h"
   s.private_header_files = "Sources/#{s.module_name}/include/FBLPromisePrivate.h"
   s.source_files = "Sources/#{s.module_name}/**/*.{h,m}"
+  s.module_map = "Sources/#{s.module_name}/include/framework.modulemap"
+  s.preserve_path = "Sources/#{s.module_name}/include/framework.modulemap"
   s.xcconfig = {
     'HEADER_SEARCH_PATHS' => "\"${PODS_TARGET_SRCROOT}/Sources/#{s.module_name}/include\""
   }
```

Comment:
Will this still work if we remove `s.xcconfig` and use `s.pod_target_xcconfig` instead?